### PR TITLE
fix: UI hard to navigate for screenreader users #3240

### LINKED ISSUE #3240: UI hard to navigate for screenreader users
Dear Backpack team,
with this issue I as a blind user would like to report a few obstacles in the UI.
* During the setup, you have to choose networks. Checkboxes would be ideal for that, since you toggle them on and off, exactly as needed here. However, buttons are used instead, making it impossible for me to see what is toggled on or off.
* Same applies in some other phases in the setup as well.
* In the main UI, most of the buttons aren't labelled at all. Some have a description below them, but this is not clear when navigating the UI with tab, since interactive HTML elements should have associated labels.
* In some points, hover-elements are used. Please replace them by their semantic HTML equivalents.
* The button to copy the public address should get the label "copied" only after it was pressed, however I see this label right when entering the UI. This is not what it meant to be, because it isn't copied before I click the button.
* As a summary: Please keep to HTML standards, and - but only - where necessary aria-attributes can be used if native HTML isn't suitable for the given use case.
I'm looking forward to seeing an accessible cryptocurrency wallet in the future!

### PR DESCRIPTION
**Pull request by staticGuru**
Closes #3240

### CODE REVIEW

File: packages/app-extension/src/components/common/WalletList.tsx [hunk 1]
```diff
@@ -808,19 +816,33 @@ function CopyButton({
...
-        e.stopPropagation();
-        setIsCopying(true);
-        setTimeout(() => setIsCopying(false), 1000);
-        onClick();
-      }}
-      inverted={inverted}
-    >
-      <>{isCopying ? "Copied!" : "Copy"}</>
-    </WalletListButtonBase>
+    <WithCopyTooltip tooltipOpen={tooltipOpen}>
```
Comment (armaniferrante) [inline packages/app-extension/src/components/common/WalletList.tsx:822]: Can we remove this change and consider it separately?

File: packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ export const BlockchainSelector = ({
                   selectedBlockchains.includes(Blockchain.SOLANA) ? (
                     <CheckBadge />
                   ) : (
-                    ""
+                    <UnCheckBadge />
```
Comment (armaniferrante) [inline packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx:84]: Can we remove this change and consider it separately?

File: packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx [hunk 3]
```diff
@@ -67,7 +67,7 @@ export const BlockchainSelector = ({
                   selectedBlockchains.includes(Blockchain.ETHEREUM) ? (
                     <CheckBadge />
                   ) : (
-                    ""
+                    <UnCheckBadge />
```
Comment (armaniferrante) [inline packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx:70]: Can we remove this change and consider it separately?

File: packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx [hunk 4]
```diff
@@ -145,8 +145,8 @@ function CheckBadge() {
       style={{
         display: "inline-block",
         position: "relative",
-        top: "4px",
-        left: "5px",
+        top: 0,
```
Comment (armaniferrante) [inline packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx:148]: Can we remove this change and consider it separately?

File: packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx (hunk 2 above)
Comment (staticGuru) [reply packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx:84]: @armaniferrante what do you mean here?
Do we need to create a separate PR for each type of change?

File: packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx (hunk 3 above)
Comment (staticGuru) [reply packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx:70]: @armaniferrante what do you mean here?
Do we need to create a separate PR for each type of change?

File: packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx (hunk 2 above)
Comment (armaniferrante) [reply packages/app-extension/src/components/Onboarding/pages/BlockchainSelector.tsx:84]: Not for each change just for the comments here.

**Pull request closed without merging**